src/test/java/com/shelterhub/service/AnimalServiceTest.java

```diff
@@ -15,18 +15,18 @@
 import java.util.UUID;
 
 import static com.shelterhub.utils.AnimalUtils.buildAnimalDTO;
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
wildcard import é melhor evitar: ex: https://www.javadude.com/posts/20040522-import-on-demand-is-evil/